sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
+1. `docker pull keystoneenclaveorg/keystone:dev`
+2. `docker run -it --entrypoint /bin/bash keystoneenclaveorg/keystone:dev --volume ABS_CERTIFIER_PATH:/certifier`
+3. `cd /keystone/build && ./scripts/run-qemu.sh`
```

Comment:
"Keystone provides a bash script to launch a RISC-V QEMU machine with a random host port forwarded to the guest SSH port (22). The port is determined at the build time, and is printed out at the beginning of the script. Note down the PORT for SSH into the emulated machine."

Reply:
resolved